usaspending_api/references/migrations/0007_auto_20170516_1702.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-05-16 17:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+from usaspending_api.references.models import LegalEntity
+
+
+def forwards_func(apps, schema_editor):
+    legalentities = apps.get_model("references", "LegalEntity")
+    for le in legalentities.objects.all():
+        LegalEntity.update_business_type_categories(le)
+        le.save()
```

Comment:
I wasn't sure if Django was smart enough to not save it because there weren't any changes :) Do you have any insight on that?